### PR TITLE
update gopkg.in/yaml.v2 to v2.2.7

### LINKED ISSUE #40: helm push not putting quotes around scientific notation like numbers
With `helm 2.16.1`, `helm-push 0.7.1` and `chartmuseum 0.10.0`, try the below

$ helm create old-chart
$ # change appVersion in Chart.yaml to "722225e2"
$ vi old-chart/Chart.yaml
$ # now push the chart to chartmuseum repo called "local"
$ helm push old-chart local
$ # check the index.yaml
$ curl http://localhost:8080/index.yaml
apiVersion: v1
entries:
  old-chart:
  - apiVersion: v1
    appVersion: "7.22225e+07"
    created: "2019-12-04T22:16:19.562137212+05:30"
    description: A Helm chart for Kubernetes
    digest: 0b891b622bd0ab3e8e22302dab504481d10887a011bb92f38830097c44b3ccd3
    name: old-chart
    urls:
    - charts/old-chart-0.1.0.tgz
    version: 0.1.0
generated: "2019-12-04T22:16:22+05:30"
serverInfo: {}

Now if you see the appVersion, you notice that it's in some scientific notation form. This does not happen when directly `curl` is used to push charts this way - 

```
$ helm package old-chart
$ # check the tar ball Chart.yaml, it will still be "722225e2" 
$ vi old-chart-0.1.0.tgz
$ curl --data-binary "@old-chart-0.1.0.tgz" http://localhost:8080/api/charts
```

The issue is in helm-push. When push is done, the chart contents are read (parsed), and then later pushed to chartmuseum. While parsing and then serializing, it removes the quotes (due to a yaml package bug) and then it pushes to the chartmusem, where the number is made into a string and put in double quotes, but it's converted a bit, compared to the original string

Updating yaml package to latest version solves the issue

### PR DESCRIPTION
**Pull request by karuppiah7890**
Fixes #40

### CODE REVIEW

**Comment by jdolitsky** [comment]
@karuppiah7890 this is blocked by https://github.com/chartmuseum/helm-push/issues/43

File: go.mod [hunk 1]
```diff
@@ -21,3 +21,5 @@ require (
 	k8s.io/client-go v10.0.0+incompatible // indirect
 	k8s.io/helm v2.13.1+incompatible
 )
+
+replace gopkg.in/yaml.v2 => gopkg.in/yaml.v2 v2.2.7
```
Comment (jdolitsky) [inline go.mod]: Is it possible to do this without a replace statement? I see its listed above on line 19

Comment (karuppiah7890) [reply go.mod]: Ah yes. I didn't realize this. I should learn go modules more 😅 I thought I could change the versions for modules that we directly depend on, and that the `// indirect` ones cannot be changed. Just now tried this, and it works with same results in `go.sum`. Making the changes! 😄 And it's better this way right? I mean I remember reading something which said that if I use replace in this repo, then any program importing this repo will not get that replace and that they will have to copy paste if they need it 😓

**Comment by karuppiah7890** [comment]
I've pushed the changes @jdolitsky 😄

**Pull request merged**